.travis.yml

```diff
@@ -6,7 +6,7 @@ python:
   - "3.5"
   - "3.6"
 install:
-  - "pip install ."
+  - "pip install .[dev]"
```

Comment:
How about 

- pip install -r requirements-dev.txt
- pip install .

Can't find a proper documentation for this [] syntax right now. Keeping simple things simple...

Reply:
Also
```
before_install:
  - pip install -U pip
```
might be an option, in case the 9.0.1 on Travis is not recent enough for that extras_require syntax.